src/prof.c

```diff
@@ -45,6 +45,8 @@ bool		opt_prof_leak = false;
 bool		opt_prof_accum = false;
 char		opt_prof_prefix[PROF_DUMP_FILENAME_LEN];
 
+prof_accum_t	prof_accum;
```

Comment:
nit: let's name this `idump_accum`. prof_accum is also the name of the struct I believe.

Reply:
Renamed it to `prof_idump_accum` to stick to the `prof_` "namespace". It clashes with the function name `prof_idump_accum()` so changing the function name to `prof_idump_increase()`.

Reply:
Hmm increase feels confusing, as in increasing interval which we don’t do.

I’d still vote for keeping the function name with accum, and name the variable maybe prof_idump_accumulated?